src/utils/helpers.ts

```diff
@@ -0,0 +1,14 @@
+import { Person } from '../types';
+
+export const getFullDetailsOfPerson = (peoples: Person[]): Person[] => {
```

Comment:
The function name `getFullDetailsOfPerson` seems to imply that it's dealing with a single person object but it's actually dealing with an array of objects. Better to name this function something like `getFullDetailsOfPeople`.